scripts/src/main/resources/scripts/command/ide

```diff
@@ -49,13 +49,15 @@ function doUpdateScripts() {
   doUpgradeMavenArtifact "${DEVON_IDE_HOME}" "${DEVON_IDE_REPO_URL}" "devonfw-ide-scripts" "${target_version}" ".tar.gz" "$[devon_ide_version]"
   if [ "${?}" = 0 ]
   then
-    if [ "${target_version}" = "LATEST" ]
+    doVersionCompare "${my_ide_version}" "${current_devon_version}"
```

Comment:
`my_ide_version` is a local variable of another function (`doUpdateCliScript`) so here it will always be empty.
IMHO all this code here is a broken duplication of existing code.
We can replace it with just a call of `doUpdateCli` and should be done.